_pytest/main.py

```diff
@@ -403,7 +403,7 @@ def _repr_failure_py(self, excinfo, style=None):
             else:
                 style = "long"
 
-        return excinfo.getrepr(funcargs=True,
+        return excinfo.getrepr(funcargs=True, abspath=True,
```

Comment:
As I said before, IMHO you will have to target `features` instead, @RonnyPfannschmidt and @The-Compiler, do you agree?

Unfortunately the only way to do that is closing this PR and re-opening it; and if in the meanwhile you could squash your commits that would be great too.